index.html

```diff
@@ -289,7 +289,7 @@ <h2 class="footer-title">Contact me</h2>
           </label>
         </li>
         <li class="form-button">
-          <button type="submit">Get in touch</button>
+          <button type="submit" id="submit-button-form">Get in touch</button>
         </li>
       </ul>
     </form>
```

Comment:
- [ ] Kindly make for your email validation when a user submits the contact form with the email field containing only lowercase characters the form is sent normally, but in your portfolio, the user can not send even messages `containing only lowercase` and messages `containing uppercase.`
Your email form behavior: 
![valid](https://user-images.githubusercontent.com/83042742/175423896-c7fd3701-6035-44a0-ba21-f9a28f9dd08b.gif)

- [ ] Kindly make the error message show up close to the "submit" button, but in your portfolio, it is displayed in the `email input` instead of at its expected required place.